documentation/scaling_puppet_server.markdown

```diff
@@ -95,17 +95,27 @@ If you [use `SRV` records for agents](#using-dns-srv-records), you can use the `
 
 To add a compile master to your deployment, begin by [installing and configuring Puppet Server](./install_from_packages.markdown) on it.
 
-In the compile master's [`puppet.conf`](https://puppet.com/docs/puppet/latest/config_file_main.html), in the `[main]` configuration block, set the [`ca_server`](https://puppet.com/docs/puppet/latest/configuration.html#caserver) setting to the hostname of the server acting as the certificate authority.
-
 Before running `puppet agent` or `puppet master` on the new server:
 
+1. In the compile master's 
+[`puppet.conf`](https://puppet.com/docs/puppet/latest/config_file_main.html), in the
+ `[main]` configuration block, set the 
+ [`ca_server`](https://puppet.com/docs/puppet/latest/configuration.html#caserver) setting
+ to the hostname of the server acting as the certificate authority.
+
+1. In the compile master's `webserver.conf` file, add and set the following SSL settings:
```

Comment:
These numbers look wrong.

Reply:
Ah, yeah, I got some formatting wonky. Lemme see if I can fix it.